modules/distribution/src/repository/resources/conf/default.json

```diff
@@ -381,128 +381,100 @@
   "identity.issuer_policy": "SelfAndManaged",
   "identity.token_validation_policy": "CertValidate",
 
-  "oauth.token.cleanup.enable": true,
-  "oauth.token.cleanup.backup_for_auditing": true,
-  "oauth.token.persist_access_token_alias": true,
+  "oauth.token_cleanup.enable": true,
+  "oauth.token_cleanup.retain_access_tokens_for_auditing": true,
+
   "oauth.use_entityid_as_issuer_in_oidc_discovery": true,
-  "oauth.authz_code.validity_time": "5m",
-  "oauth.app_token.validity_time": "1h",
-  "oauth.user_token.validity_time": "1h",
-  "oauth.refresh_token.validity_time": "1410m",
+  "oauth.token_validation.authorization_code_validity": "5m",
+  "oauth.token_validation.app_access_token_validity": "1h",
+  "oauth.token_validation.user_access_token_validity": "1h",
+  "oauth.token_validation.refresh_token_validity": "1410m",
```

Comment:
isn't this 1440m? Let's have it 1d instead. (easiest to interpret for a human)

Reply:
The original value is 84600s. When converted to minutes it is 1410m. It will be like 23.5h. Is it okay to round up? There is no mandatory value specified for refresh token validity in the OAuth2 specification as well

Reply:
Should have intended 86400 (which is 1 day), instead of 84600. So, let's change it to 1d.

Reply:
Fixed.